sympy/utilities/codegen.py

```diff
@@ -1204,7 +1204,7 @@ def _call_printer(self, routine):
                 assign_to = result.result_var
 
             constants, not_fortran, f_expr = self._printer_method_with_settings(
-                'doprint', dict(human=False, source_format='free'),
+                'doprint', dict(human=False, source_format='free', standard=95),
```

Comment:
It is not clear to me if the f2py call enforces the use of the F95 compiler. Anybody know?